migration/sql-files/028-iterations-parent-path-ltree.sql

```diff
@@ -0,0 +1,22 @@
+CREATE EXTENSION IF NOT EXISTS "ltree";
+
+-- Rename parent_id column
+ALTER TABLE iterations RENAME parent_id to path;
+
+-- Need to convert the path column to text in order to
+-- replace non-locale characters with an underscore
+ALTER TABLE iterations ALTER path TYPE text USING path::text;
+
+-- Need to update values of Iteration's' ParentID in order to migrate it to ltree
+-- Replace every non-C-LOCALE character with an underscore
+UPDATE iterations SET path = regexp_replace(path, '[^a-zA-Z0-9_\.]', '_', 'g');
```

Comment:
Hi @sbose78 : I have checked with all spaces; all related iterations using APIs to see if there is any nested iteration. I found that there is no nested iteration as of now. So we can go ahead with this PR without any changes IMO.